de.metas.swat/de.metas.swat.base/src/main/java/de/metas/inoutcandidate/api/IShipmentScheduleEffectiveBL.java

```diff
@@ -73,7 +77,8 @@ default int getC_BP_Location_ID(final I_M_ShipmentSchedule sched)
 
 	I_AD_User getAD_User(I_M_ShipmentSchedule sched);
 
-	int getAD_User_ID(I_M_ShipmentSchedule sched);
+	@Nullable
+	BPartnerContactId getADUserID(@NonNull I_M_ShipmentSchedule sched);
```

Comment:
IMHO u can call it `getBPartnerContactId`

Reply:
done